src/main/groovy/org/labkey/gradle/task/DeployAppBase.groovy

```diff
@@ -54,6 +54,7 @@ class DeployAppBase extends DefaultTask {
                 project.copy { CopySpec copy ->
                     copy.from dir
                     copy.into deployBinDir.getPath()
+                    copy.setDuplicatesStrategy(DuplicatesStrategy.EXCLUDE)
```

Comment:
I'm leaving this as EXCLUDE since that's what's used in the `if` statement above this copy (used only on TC).